### ISSUE TITLE
bunx shadcn-svelte init fails

### ISSUE THREAD

**Issue by Kazumasan** [oven-sh/bun#12208]
### How can we reproduce the crash?

run `bunx shadcn-svelte@latest init`
in an empty sveltekit skeleton project. 

If you think this issue is related to something else, close it.

### Relevant log output

```shell
============================================================                                                            Bun v1.1.10 (5102a944) Windows x64                                                                                      Args: "bun" "C:\Users\Bastian\AppData\Local\Temp\bunx-2969596043-shadcn-svelte@latest\node_modules\shadcn-svelte\dist\index.js" "init"                                                                                                          Features: jsc Bun.stdin(2) Bun.stdout fetch spawn(2) transpiler_cache(3) tsconfig_paths tsconfig(2)                     Builtins: "bun:main" "node:assert" "node:async_hooks" "node:buffer" "node:child_process" "node:console" "node:crypto" "node:events" "node:fs" "node:http" "node:https" "node:module" "node:net" "node:os" "node:path" "node:perf_hooks" "node:process" "node:querystring" "node:readline" "node:stream" "node:string_decoder" "node:tls" "node:tty" "node:url" "node:util" "node:util/types" "node:zlib" "node:worker_threads" "node:http2" "node:diagnostics_channel"                          Elapsed: 8785ms | User: 78ms | Sys: 0ms                                                                                 RSS: 0.27GB | Peak: 0.27GB | Commit: 4.64GB | Faults: 68720                                                                                                                                                                                     panic(main thread): Segmentation fault at address 0x5                                                                   oh no: Bun has crashed. This indicates a bug in Bun, not your code.                                                                                                                                                                             To send a redacted crash report to Bun's team,                                                                          please file a GitHub issue using the link below:
```

### Stack Trace (bun.report)

Bun v1.1.10 ([`5102a94`](<https://github.com/oven-sh/bun/tree/5102a944307d56933a6bb039b99ad76a758b46e4>)) on windows x86_64 [AutoCommand]

Segmentation fault at address 0x00000005

- *5 unknown/js code*
- `??`
- *4 unknown/js code*

**Comment by Kazumasan** [comment]
that headsup is nice. 

yes issue persists: 

```md
Bun v1.1.17 (bb66bba1) Windows x64
Args: "bun" "C:\Users\Bastian\AppData\Local\Temp\bunx-2969596043-shadcn-svelte@latest\node_modules\shadcn-svelte\dist\index.js" "init"
Features: jsc Bun.stdin(2) Bun.stdout dotenv fetch spawn(2) tsconfig_paths tsconfig(2) 
Builtins: "bun:main" "node:assert" "node:async_hooks" "node:buffer" "node:child_process" "node:console" "node:crypto" "node:events" "node:fs" "node:http" "node:https" "node:module" "node:net" "node:os" "node:path" "node:perf_hooks" "node:process" "node:querystring" "node:readline" "node:stream" "node:string_decoder" "node:tls" "node:tty" "node:url" "node:util" "node:util/types" "node:zlib" "node:worker_threads" "node:http2" "node:diagnostics_channel" 
Elapsed: 7018ms | User: 62ms | Sys: 0ms
RSS: 0.29GB | Peak: 0.29GB | Commit: 4.69GB | Faults: 75385

panic(main thread): Segmentation fault at address 0x5
oh no: Bun has crashed. This indicates a bug in Bun, not your code.

To send a redacted crash report to Bun's team,
please file a GitHub issue using the link below:
```

**Comment by nektro** [comment]
```
┌   shadcn-svelte  v0.14.0 
│
▲  You are currently using an unsupported runtime. Only Node.js v18 or higher is officially supported. Continue at your own risk.
│
└  ENOENT: No such file or directory
    at readFileSync (native)
    at readFileSync (native)
    at Wt (/private/tmp/bunx-501-shadcn-svelte@latest/node_modules/shadcn-svelte/dist/index.js:58:3791)
    at Ji (/private/tmp/bunx-501-shadcn-svelte@latest/node_modules/shadcn-svelte/dist/index.js:58:4072)
    at <anonymous> (/private/tmp/bunx-501-shadcn-svelte@latest/node_modules/shadcn-svelte/dist/index.js:58:3870)
    at asyncFunctionResume (native:1:11)
    at on (/private/tmp/bunx-501-shadcn-svelte@latest/node_modules/shadcn-svelte/dist/index.js:58:3864)
    at <anonymous> (/private/tmp/bunx-501-shadcn-svelte@latest/node_modules/shadcn-svelte/dist/index.js:239:1674)
    at asyncFunctionResume (native:1:11)
    at Vs (/private/tmp/bunx-501-shadcn-svelte@latest/node_modules/shadcn-svelte/dist/index.js:239:1661)
```

**Comment by heimskr** [comment]
Probably fixed by #15357.
